src/Block/Service/EmptyBlockService.php

```diff
@@ -18,22 +18,11 @@
 use Sonata\BlockBundle\Model\BlockInterface;
 use Sonata\CoreBundle\Validator\ErrorElement;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class EmptyBlockService extends AbstractBlockService
 {
     // NEXT_MAJOR: Remove this method
 
-    /**
-     * {@inheritdoc}
-     */
-    public function configureSettings(OptionsResolver $resolver): void
```

Comment:
I am trying to find where this was removed on 3.x, can't really find. Can you show me?

Reply:
That's probably an error on my part